scipy/signal/tests/test_spectral.py

```diff
@@ -969,6 +969,68 @@ def test_amplitude(self):
         # frequency is less than accuracy
         assert_approx_equal(np.max(pgram), ampl, significant=2)
 
+    def test_precenter(self):
+        """Test if precenter gives the same result as manually precentering.
```

Comment:
The convention in scipy is to use comments instead of docstrings in test functions. Can you change it please?